src/index.js

```diff
@@ -280,8 +280,11 @@ class ReactPhoneInput extends React.Component {
     });
   }
 
-  handleInputClick() {
+  handleInputClick(evt) {
     this.setState({showDropDown: false});
+    if (this.props.onClick) {
+      this.props.onClick(evt)
```

Comment:
Yeah, sure, but I wonder about the readability and I think it's not really important.